docs/source/installer/packstack.rst

```diff
@@ -0,0 +1,44 @@
+Packstack
+---------
+
+Infrared allows to use the Packstack to install Openstack::
+
+    $ ir-installer -d -vvvv --inventory hosts packstack --product-version=8 -o install.yml -e @provision.yml -e @private.yml
+
+Here required arguments are:
+    --product-version: the product version to install.
+
+Optional arguments:
+    * provision.yml - the settings file generated ny provisiner. I might contains required for Packstack settings.
```

Comment:
the settings file generated by provisiner (using `ir-provisioner [...] -o provision.yml [...]`). It might contain relevant data required for Packstack settings.